src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -67,7 +67,7 @@ const perPageKey = 'snapshotPerPage';
 export default function SnapshotListModal() {
   const classes = useStyles();
   const rootPath = useRootPath();
-  const { repoUUID: uuid = '' } = useParams();
+  const { repoUUID: uuid, snapshotUUID = '' } = useParams();
```

Comment:
as mentioned, no need to add snapshotUUID here as it won't be accessible on [this](https://github.com/content-services/content-sources-frontend/pull/207/files#diff-21c0871aec2cc959996fa3600d513fccb3575d5d72dd892b0102df47b62058acR44) route.